File: python/7kyu/highest_and_lowest.py
# https://www.codewars.com/kata/554b4ac871d6813a03000035

def high_and_low(numbers):
    return str(max(n := [int(i) for i in numbers.split()])) + ' ' + str(min(n))


print(high_and_low("8 3 -5 42 -1 0 0 -9 4 7 4 -4"), "42 -9")

File: go/7kyu/highest_and_lowest.go
// https://www.codewars.com/kata/554b4ac871d6813a03000035

package main

import (
	"fmt"
	"math"
	"strconv"
	"strings"
)

func HighAndLow(in string) string {
	min, max := math.MaxInt, math.MinInt
	for _, n := range strings.Fields(in) {
		intN, _ := strconv.Atoi(n)
		if intN < min {
			min = intN
		} else if intN > max {
			max = intN
		}
	}

	return fmt.Sprintf("%d %d", max, min)
}

func main() {
	fmt.Println(HighAndLow("1 9 3 4 -5"), "9 -5")
}
